platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/Extensions/NSAttributedString+Range.swift

```diff
@@ -148,7 +132,7 @@ extension NSAttributedString {
                 }
             }
         }
-
+                        
```

Comment:
Seems like you have one of the two `automatically trim whitespaces` options disabled ?

Reply:
yeah had the` including whitespace only lines` one disabled